apps/api/src/app/topics/use-cases/create-topic/create-topic.use-case.ts

```diff
@@ -23,9 +28,21 @@ export class CreateTopicUseCase {
       throw new ConflictException(`There is already a topic with the key ${entity.key} for user ${entity._userId}`);
     }
 
-    const topic = await this.topicRepository.createTopic(entity);
+    const topicDb = await this.topicRepository.createTopic(entity);
+    const topic = this.mapFromEntity(topicDb);
+
+    const createTopicSubscribersCommand = CreateTopicSubscribersCommand.create({
+      environmentId: command.environmentId,
+      organizationId: command.organizationId,
+      topicId: topic._id,
+      userId: command.userId,
+    });
+    const topicSubscribers = await this.createTopicSubscribersUseCase.execute(createTopicSubscribersCommand);
 
-    return this.mapFromEntity(topic);
+    return {
+      ...topic,
+      subscribers: topicSubscribers.subscribers,
```

Comment:
This will be empty anyway but building the DTO returned in this way in case the logic underneath changes.